querydsl-apt/src/main/java/com/querydsl/apt/ExtendedTypeFactory.java

```diff
@@ -34,7 +34,7 @@
  * @author tiwe
  *
  */
-final class ExtendedTypeFactory {
+public class ExtendedTypeFactory {
```

Comment:
Why the change?

Reply:
Similar to previous comment - https://github.com/infobip/infobip-spring-data-querydsl/pull/9/files#diff-e45095386f2afae8a4499b6f48a71368